### PR TITLE
Added Stage 5 Explainer, Updated CRDS env vars

### PR DESCRIPTION
**Pull request by jbrande**
Added initial explainer for Stage 5 ECFs, still need explanations of the various transit/eclipse and systematics models. Added more info to clarify setting the CRDS environment variables.

### CODE REVIEW

File: docs/source/ecf.rst [hunk 1]
```diff
@@ -406,3 +406,150 @@ The path to the directory containing the Stage 3 JWST data.
...
+``Name    Value    Free    PriorPar1    PriorPar2    PriorType``
+
+The ``PriorType`` can be U (Uniform), LU (Log Uniform), or N (Normal). If U/LU, then ``PriorPar1`` and ``PriorPar2`` are the lower and upper limits of the prior distribution. If N, then ``PriorPar1`` is the mean and ``PriorPar2`` is the stadard deviation of the Gaussian prior.
+
+``Free`` determines whether the parameter is fixed, free, independent, or shared. Fixed parameters are fixed in the fitting routine and not fit for. Free parameters are fit for according to the specified prior distribution, independently for each wavelength channel. Shared parameters are fit for according to the specified prior distribution, but are common to all wavelength channels. Independent variables set auxiliary functions needed for the fitting routines.
+
+Here's an example fit parameter file:
+
+
+.. include:: ../media/S5_fit_par_template.ecf
```
Comment (taylorbell57) [inline docs/source/ecf.rst:554]: If there is still the transittype line in that fitpar template (and the one in demos) can you please delete it

**Review by taylorbell57 — CHANGES_REQUESTED** [review]

File: docs/source/ecf.rst (hunk 1 above)
Comment (jbrande) [reply docs/source/ecf.rst:554]: Ah, yeah. It's still there. Deleting now.

File: docs/source/ecf.rst [hunk 2]
```diff
@@ -406,3 +406,150 @@ The path to the directory containing the Stage 3 JWST data.
...
+Integer. The number of burn-in steps to run.
+
+
+Dynesty Fitting Parameters
+''''''''''''''''''''''''''
+The following set the parameters for running dynesty. These options are described in more detail in: https://dynesty.readthedocs.io/en/latest/api.html?highlight=unif#module-dynesty.dynesty
+
+run_nlive
+'''''''''
+Integer. Number of live points for dynesty to use.
```
[the remark this answers is not in the document]
Comment (jbrande) [reply docs/source/ecf.rst:491]: Got it.

**Review by taylorbell57 — APPROVED** [review]
In the near future we should also explain what all of the potential fitted parameters are in S5, but that doesn't need to be in this PR

**Pull request merged**